# Feature coordinages in left image, measured from top-left corner
float64 left_x
float64 left_y
# Feature coordinages in right image, measured from top-left corner
float64 right_x
float64 right_y
# Feature id (optional)
int64 id
